src/node.cc

```diff
@@ -2509,9 +2494,15 @@ void FatalException(Isolate* isolate,
     // Do not call FatalException when _fatalException handler throws
     fatal_try_catch.SetVerbose(false);
 
+    Local<Value> argv[2] = {
+        error,
+        Boolean::New(env->isolate(), should_catch)
+    };
+
     // this will return true if the JS layer handled it, false otherwise
-    Local<Value> caught =
-        fatal_exception_function->Call(process_object, 1, &error);
+    Local<Value> caught = fatal_exception_function->Call(process_object,
+                                                         2,
+                                                         argv);
```

Comment:
If you’re changing this code anyway, you can feel free to switch to the non-deprecated version of `Call()`? I think the easiest version would be to add `context` as a first argument and use `.FromMaybe(Local<Value>())` to the return value.

(You can’t use `.ToLocalChecked()` here because we do check for an exception below).